src/controllers/factorycontroller.js

```diff
@@ -32,6 +32,18 @@ class FactoryController extends OrderController
             }
         );
 
+        //adds an extra truck after cost validation
+        $("#buy-truck").click(() => {
+            if(GAME.model.config.money > GAME.model.config.costExtraTruck){
```

Comment:
`if (...) {` whitespaces! I know this is a little pedantic, but a uniform style has its merits.